orchestrator/cli/database.py

```diff
@@ -239,6 +240,62 @@ def remove_core_as_down_revision(migration: Any) -> None:
         f.write(text)
 
 
+def _insert_preamble(text: str, s: str) -> str:
```

Comment:
move this function and the one below to a helper file, so we only keep command fuctions here.

would be good to create `helper` folder with generic helpers, there are some generic functions in `domain_gen_helpers`, `print_helpers` for example, that could also be moved to the generic `helper` folder.

Reply:
There is now a "migration_helpers.py". It's not very generic, but just improves code organization